decidim-core/app/cells/decidim/card_g_cell.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This cell is used a base for all Grid cards. It holds the basic layout
+  # so other cells only have to customize a few methods or overwrite views.
+  class CardGCell < Decidim::ViewModel
+    include Decidim::ApplicationHelper
+    include Decidim::SanitizeHelper
+
+    alias resource model
+
+    def show
+      render
+    end
+
+    private
+
+    def highlight?
+      options[:highlight] == true
+    end
+
+    def classes
+      # avoid metaprogramming in CSS classes due to Tailwind purge
```

Comment:
I am not exactly sure how tailwind works, but you could split this in 2 different methods ?

```
def classes
  @classes ||= if highlight? ? highlight_classes : grid_classes
end 

def highlight_classes
{
                       default: "card__highlight",
                       img: "card__highlight-img",
                       text: "card__highlight-text",
                       metadata: "card__highlight-metadata"
                     }
end
....
```

O even better, you could use 2  constants to keep the classes 
```
HIGHLIGHT_CSS = {
                       default: "card__highlight",
                       img: "card__highlight-img",
                       text: "card__highlight-text",
                       metadata: "card__highlight-metadata"
                     }.freeze 
DEFAULT_CSS =                        default: "card__grid",
                       img: "card__grid-img",
                       text: "card__grid-text",
                       metadata: "card__grid-metadata"
                     }.freeze 
def classes
  @classes ||= highlight? ? HIGHLIGHT_CSS : DEFAULT_CSS
end 
```

Reply:
You are right, I think it will work fine using constants